### DISCUSSION TITLE
Add a features to have Toggle Headings like in Notion or Quip

### DISCUSSION THREAD

**Discussion by ChristopherHButler** [shuding/nextra#3128] [Ideas]
I would first just like to say I am absolutely LOVING nextra, thank you so much for all your hard work!!!

I was wondering if if was on the roadmap to develop a feature where a heading could be toggled similar to Toggle Headings in Notion or the default behavior in Quip. If you are not familiar with either of those it's basically the ability to have code folding but for headings. 

I know you can do this with standard markdown but the result and mdx file ends up a bit messy having indentation and borders. 

Anyone planning something towards this feature?

**Comment by Takib-Ahmed** [comment]
yeah I was a bit disspointed to not have toogles like notion after exporting the markdown from there. Now I have to make a custom one for me
